orchagent/portsorch.cpp

```diff
@@ -21,6 +21,43 @@ extern MacAddress gMacAddress;
 
 #define FRONT_PANEL_PORT_VLAN_BASE 1024
 
+namespace swss {
+sai_object_id_t Port::getQueue(size_t queue_ind)
```

Comment:
what about return boolean and put queue_id into arguments?
bool Port::getQueue(size_t queue_ind, sai_object_id_t &queue_id)